src/Commands/MissingCommand.php

```diff
@@ -169,6 +174,15 @@ private function getMissing(array $languages)
 
         $missing = array_merge($missing, $this->manager->getKeysExistingInALanguageButNotTheOther($values));
 
+	    if ( ! empty($languages)) {
```

Comment:
@mustafaaloko 
Thanks for your help, removed the spaces as you mentioned, do you have any idea why CI failing in the tests ?